tsconfig.json

```diff
@@ -25,7 +25,6 @@
         "noUnusedLocals": true, // Report errors on unused locals
         "noUnusedParameters": true, // Report errors on unused parameters
         "experimentalDecorators": true, // Enables experimental support for ES decorators
-        "incremental": true, // Enable incremental compilation by reading/writing information from prior compilations to a file on disk
```

Comment:
Context, what was the error?

Reply:
With `noEmit: true` this flag doesn't make sense. 
- noEmit - no output
- incremental - incremental output, e.g. during development

Since we're using TSC just to check the types and babel-typescript for the actual processing in webpack, the right one to keep is `noEmit`